docs/source/autograd.rst

```diff
@@ -117,3 +117,12 @@ Anomaly detection
 .. autoclass:: detect_anomaly
 
 .. autoclass:: set_detect_anomaly
+
+
+Context method mixins
```

Comment:
I would prefer to add this line 84 of this file, just after the definition of `Function`. What do you think?

Reply:
Agree.